core/state/statedb_test.go

```diff
@@ -32,14 +32,18 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/rawdb"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/trie"
 )
 
 // Tests that updating a state trie does not leak any database writes prior to
 // actually committing the state.
 func TestUpdateLeaks(t *testing.T) {
 	// Create an empty state database
-	db := rawdb.NewMemoryDatabase()
-	state, _ := New(common.Hash{}, NewDatabase(db), nil)
+	var (
+		db  = rawdb.NewMemoryDatabase()
+		tdb = trie.NewHashDatabase(db)
```

Comment:
If I replace tbd with a path database here I get the following error:
```
statedb_test.go:63: can not commit trie 0x1b7916b2e01f0449a62343afe2b4bed86186ada4acfe628cb9f9bcabe33ed425 to persistent database, err: triedb snapshot [0x1b7916b2e01f0449a62343afe2b4bed86186ada4acfe628cb9f9bcabe33ed425] missing
```
I'm pretty sure that this is something that can be ignored, but I wanted to make sure